governance/nested-governance-processing.php

```diff
@@ -81,37 +81,40 @@ public static function get_nested_settings_and_css( $governance_rules ) {
 	 * @param array $nodes             The metadata of the nodes that have been built so far.
 	 * @param array $current_selector  The current selector of the current block.
 	 * @param array $current_path      The current path to the block.
-	 * 
-	 * @return array 
+	 *
+	 * @return array
 	 */
 	private static function get_settings_of_blocks( $blocks_registered, $current_block, $nodes = [], $current_selector = null, $current_path = [] ) {
 		foreach ( $current_block as $block_name => $block ) {
-			if ( array_key_exists( $block_name, $blocks_registered ) ) {
+			if ( ! array_key_exists( $block_name, $blocks_registered ) && ! str_ends_with( $block_name, '/*' ) ) {
+				continue;
+			}
 
-				$selector = is_null( $current_selector ) ? null : $current_selector;
+			$selector = is_null( $current_selector ) ? null : $current_selector;
 
-				// This function is only available in 6.3 and above.
-				if ( function_exists( ( 'wp_get_block_css_selector' ) ) ) {
-					$looked_up_selector = wp_get_block_css_selector( $blocks_registered[ $block_name ] );
-				} else {
-					// Once the 6.3 upgrade is done, this will be deprecated.
-					$looked_up_selector = self::get_css_selector_for_block( $block_name, $blocks_registered );
-				}
+			// If the block name ends with /*, then it's a regex rule and we need to use the wp-block selector to match against any block.
+			if ( str_ends_with( $block_name, '/*' ) ) {
+				$looked_up_selector = '.wp-block';
+			} elseif ( function_exists( ( 'wp_get_block_css_selector' ) ) ) {
+				$looked_up_selector = wp_get_block_css_selector( $blocks_registered[ $block_name ] );
+			} else {
+				// Once the 6.3 upgrade is done, this will be deleted.
```

Comment:
Is this comment still meaningful?

Reply:
Yip, once we set the min WP version to 6.3 then this would no longer be necessary. The new selector would be sufficient

Reply:
Got it, I wasn't sure what that meant even though it's obvious now. Can we add that in:

```suggestion
				// Once our minimum WordPress version >= 6.3, this can be deleted.
```